README.md

```diff
@@ -1 +1,9 @@
-# vamp-health-probe
\ No newline at end of file
+# Vamp Health Probe
+
+This repository contains helper structs to run Kubernetes health checks in parallel.
+
+## Running unit tests
+
+```bash
+make test
+```
```

Comment:
Only minor comment: perhaps it'd be good to show an example how to use / implement the helper

Reply:
Good point, I added an example.